src/components/FilterList/FilterComponent.tsx

```diff
@@ -46,37 +43,6 @@ export abstract class Component<P extends Props<R>, S extends State<R>, R> exten
     this.handleError(errMsg);
   }
 
-  pageSet = (page: number) => {
-    this.setState(prevState => {
-      return {
-        listItems: prevState.listItems,
-        pagination: {
-          page: page,
-          perPage: prevState.pagination.perPage,
-          perPageOptions: ListPagesHelper.perPageOptions
-        }
-      };
-    });
-    HistoryManager.setParam(URLParam.PAGE, String(page));
-  };
-
-  perPageSelect = (perPage: number) => {
-    this.setState(prevState => {
-      return {
-        listItems: prevState.listItems,
-        pagination: {
-          page: 1,
-          perPage: perPage,
-          perPageOptions: ListPagesHelper.perPageOptions
-        }
-      };
-    });
-    HistoryManager.setParams([
-      { name: URLParam.PAGE, value: '1' },
-      { name: URLParam.PER_PAGE, value: String(perPage) }
```

Comment:
These params that are not used anymore can be removed from the enum (in History.ts)